homeassistant/components/denonavr/media_player.py

```diff
@@ -256,7 +256,7 @@ async def _telnet_callback(self, zone, event, parameter) -> None:
             return
         # Some updates trigger multiple events like one for artist and one for title for one change
         # We skip every event except the last one
-        if event == "NS" and not parameter.startswith("E4"):
+        if event == "NSE" and not parameter.startswith("4"):
             return
         if event == "TA" and not parameter.startwith("ANNAME"):
             return
```

Comment:
It would be nice if HA knew less about these but I don't have a specific suggestion here since I suspect these are going to be HA specific.